docs/content/Workspace/SQL-Runner.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: SQL Runner
+permalink: /cloud/sql-runner
+category: Workspace
+menuOrder: 6
+---
+
+The SQL Runner is a tool that allows you to execute SQL queries from Cube Cloud
+on your data source or Cube Store. It can be used for ad-hoc querying as well as
+debugging SQL queries generated by Cube to execute against the data source.
+
+<SuccessBox>
+
+The SQL Runner is available in Cube Cloud. [Click here][cubecloud-signup] to
+sign up for a new account.
```

Comment:
<img width="907" alt="Screenshot 2023-04-26 at 14 50 05" src="https://user-images.githubusercontent.com/3852894/234579918-a1f21b92-6bf5-4135-85f9-87e81c8fbbf3.png">
Let's follow this pattern for Cloud features available on all tiers. It would be easier to replace these with a custom banner later.